### ISSUE TITLE
large request? (250mb) causes crash? not sure

### ISSUE THREAD

**Issue by kelvinwop** [oven-sh/bun#12733]
### How can we reproduce the crash?

```
import axios from 'axios';

async function getRaydiumData() {
  let config = {
    method: 'get',
    maxBodyLength: Infinity,
    url: 'https://api.raydium.io/v2/main/pairs',
    headers: {
    }
  };
  const response = await axios.request(config);
  console.log(response.data.at(0));
}
```

getRaydiumData();

### Stack Trace (bun.report)

Bun v1.1.20 ([`ae19489`](<https://github.com/oven-sh/bun/tree/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66>)) on windows x86_64 [RunAsNodeCommand]

Segmentation fault at address 0x1E3DFD8EB30

- [`memcpy.asm:50`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/src/src/string/amd64/memcpy.asm#L50>): `memcpy_repmovs`
- [`js_brotli.zig:573`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/src/bun.js/api/js_brotli.zig#L573>): `run`
- [`js_brotli.zig:526`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/src/bun.js/api/js_brotli.zig#L526>): `decodeSync`
- [`JSCJSValue.h:746`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/C:/a/bun/bun/build/bun-webkit/include/JavaScriptCore/JSCJSValue.h#L746>): `JSC::ensureStillAliveHere`
- *2 unknown/js code*
- `llint_entry`

**Comment by kelvinwop** [comment]
exact same code runs fine with nodejs as .mjs file, fails with bun

**Comment by dylan-conway** [comment]
Fixed in bun v1.1.21 with #12800
